docs/REST API/rest-api.md

```diff
@@ -9941,20 +9941,20 @@ List available Fleet-maintained apps.
     {
       "id": 1,
       "name": "1Password",
-      "version": "8.10.40",
-      "platform": "darwin"
+      "platform": "darwin",
+      "software_title_Id": 1
```

Comment:
```suggestion
      "software_title_id": 1
```

Lowercase `id` right? I think that's consistent with the rest of the API.

Reply:
Yep, not sure how that got in there. Will fix.